utils/Game.py

```diff
@@ -1,39 +1,41 @@
-from utils.Logger import Logger
+from utils.ScoreLogger import ScoreLogger
 import gym
 
 class Game:
-    def __init__(self, steps, strategy):
-        self.steps = steps
+    def __init__(self, episodes, strategy, render=False):
+        self.episodes = episodes
         self.strategy = strategy
-        self.logger = Logger()
+        self.logger = ScoreLogger()
+        self.render = render
 
-        self.env = gym.make('CartPole-v0')
-        self.env.reset()
+        self.env = gym.make('CartPole-v1')
 
-    def log(self, observation):
-        self.logger.log(
-            self.strategy.get_pole_position(observation),
-            self.strategy.get_pole_velocity(observation),
-            self.strategy.get_pole_angle(observation),
-            self.strategy.get_pole_tip_velocity(observation)
-        )
-
-    def print_logs(self):
-        print(self.logger.get_positions())
-        print(self.logger.get_velocities())
-        print(self.logger.get_angles())
-        print(self.logger.get_tip_velocities())
+    def log(self, score):
+        self.logger.log(score)
```

Comment:
Wouldn't we want to log all of the observations so that we can have more data to analyze in our report?

Reply:
Added it back in.